cfme/tests/infrastructure/test_vmware_provider.py

```diff
@@ -66,9 +71,33 @@ def test_appliance_scsi_control_vmware():
         casecomponent: Appliance
         caseimportance: critical
         initialEstimate: 1/4h
-    #TODO: yet to test this, once done, I will add steps. Test was not written by me originally.
     """
-    pass
+    url = ("{baseurl}/{series}/{ver}/cfme-vsphere-paravirtual-{ver}-1.x86_64.vsphere.ova"
+        .format(baseurl=conf.cfme_data['basic_info']['cfme_images_url']['baseurl'],
+            series=appliance.version.series(), ver=appliance.version))
+    logger.info("Downloading ova file for parvirtual vsphere scsi controller test from %s" % url)
```

Comment:
This test needs to be marked long-running, this download could take quite a while.